packages/core/src/document/index.tsx

```diff
@@ -27,7 +27,9 @@ export interface DocumentProps {
 }
 
 export function isDocumentIdentifier(
-  state: DocumentProps['state']
+  state: unknown
 ): state is DocumentIdentifier {
-  return (state as DocumentIdentifier).$$typeof !== undefined
+  return (
+    state !== undefined && (state as DocumentIdentifier).$$typeof !== undefined
```

Comment:
why was this change necessary?

Reply:
So you can also apply this to `null` values.